src/lib/repl_service_ctx.cpp

```diff
@@ -77,6 +77,29 @@ const std::string& repl_service_ctx::raft_leader_id() const {
     return empty;
 }
 
+std::vector< peer_info > repl_service_ctx::get_raft_status() const {
+    std::vector< peer_info > peers;
+    if (!is_raft_leader()) return peers;
+    if (!_server) return peers;
+
+    auto pinfo_all = _server->get_peer_info_all();
```

Comment:
Was there a reason to not use cluster configuration? Does it not have these stats too and include the leader?
Oh, it's missing the latency maybe?

Reply:
srv_config is more the static info but we are looking for dynamic info , i.e the latest commited index in any members.  

We cannot take a raft group with 3 members as "healthy" if a member is significantly laggy behind, should look it as 2 healthy replica.   CM/SRE need to do something and also GW can avoid sending read to the laggy member.